src/scales/scale.time.js

```diff
@@ -36,10 +40,94 @@ module.exports = function(Chart) {
 			},
 		},
 		ticks: {
-			autoSkip: false
+			autoSkip: false,
+			mode: 'series',   // 'linear|series'
```

Comment:
`auto` doesn't exist in 2.6.0. It's been added since the most recent release. In fact it's a breaking change to switch it to `auto` since the only behavior that exists in 2.6.0 is `linear`

`series` & `auto` together is not doing something that makes a lot of sense to me. Perhaps we should disallow that combination of options. If `auto` is forcing it to be `linear` eventhough the user specified `series` that's going to be really confusing